src/app/test_executive/zkapps.ml

```diff
@@ -304,6 +304,45 @@ module Make (Inputs : Intf.Test.Inputs_intf) = struct
       in
       return @@ Transaction_snark.For_tests.multiple_transfers parties_spec
     in
+    let%bind.Deferred parties_update_timing =
```

Comment:
this has become a really long test and it would definitely be good if you could break this out into `section`s

Reply:
There are already several sections. The code to create the individual `Parties.t` is quick, it's the sections that take all the time.

I was considering taking the timing-related sections into a new test file, does that make sense?

Reply:
ok sure